Lib/webbrowser.py

```diff
@@ -604,6 +657,14 @@ def main():
     for o, a in opts:
         if o == '-n': new_win = 1
         elif o == '-t': new_win = 2
+        elif o == "-i":
+            new_win = 3
+        elif o == "-b":
```

Comment:
Why do we have `-b` and `-l` added? I am not opposed to this, but again: this feature is not related. 

If you feel like adding this feature is a good idea (it looks like it might be), please open a new issue: explain the use-case and the "why".

But, let's keep this PR simple: only `incognito` mode.

Reply:
it is related to the cmdline interface  when webbroser main() func is called.
they belong also to the newly added incognito method.
therefore it is not a new feature from my perspective.

Reply:
This is not resolved at all.  These are separate new additions.

Reply:
Agreed

Reply:
feel free to split that and follow up

Reply:
We do appreciate your contribution here!
As the PR author, you are in the best place to address the review by removing the code added to support `--browser`.
That can be discussed in a separate issue.  But if you don’t have the time, then someone else could complete the PR.

Reply:
for me this is a migration topic. the experts do know better what to do here. myself i dont see here in the working loop since i dont have this detailed knowledge of what being best on master code baseline

Reply:
i just added you all to this repo.
do let me know in case further authorization is missing.

Reply:
Sorry, I don’t understand «migration».  There are clear requests to remove some code that was added, to keep this PR focused on one thing.  If you don’t want to pursue it, then someone else will have to volunteer.